src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -284,6 +308,7 @@ private boolean matches(Offer offer, Request request) {
           "\nRequested for Jenkins slave:\n" +
           "  cpus: " + requestedCpus + "\n" +
           "  mem:  " + requestedMem + "\n" +
+          "  ports: " + hasPortMappings + "\n" +
```

Comment:
@barkerd427 True that. Realized this when I started a slave with the VNC port exposed and I had no clue to which host port I have to connect. In the meantime I've implemented a feature which shows the (host) ports exposed on the Mesos Slaves on the Jenkins Slave Status page. Still, I have to test this and prepare another pull request...